src/components/grid/types.ts

```diff
@@ -0,0 +1,71 @@
+export const breakpoints = {
+  mobile: 'mobile',
+  mobileLg: 'mobile-lg',
+  tablet: 'tablet',
+  tabletLg: 'tablet-lg',
+  desktop: 'desktop',
+  desktopLg: 'desktop-lg',
+  widescreen: 'widescreen',
+}
+
+export type Breakpoints =
+  | 'card'
```

Comment:
Hm you're right this isn't being used anywhere (there was a lot of back and forth on this so that's probably where this came from). I excluded handling the `card` breakpoints because they're not explicitly called out at https://designsystem.digital.gov/utilities/layout-grid/ and they don't seem to have any associated media queries. I'll remove this type